tests/queries/0_stateless/02862_sorted_distinct_sparse_fix.reference

```diff
@@ -10,3 +10,4 @@ set max_threads=1;
 select trimLeft(explain) from (explain pipeline SELECT DISTINCT id, v FROM t_sparse_distinct) where explain ilike '%DistinctSortedChunkTransform%';
 DistinctSortedChunkTransform
 SELECT DISTINCT id, v FROM t_sparse_distinct format Null;
+DROP TABLE t_sparse_distinct;
```

Comment:
I'd rather add `-- { echoOff }` in sql file before `DROP` statement

Reply:
Let's merge to fix fasttest faster